### ISSUE TITLE
how to setContentInfo for push notification

### ISSUE THREAD

**Issue by nhoctanker** [phonegap/phonegap-plugin-push#957]
hi i have a issue , I don't know how to set a text in right notification 
My friend tell me in java we have method to do that. , that is  method setContentInfo
Please tell me. your plugin can setContentInfo with params from payload or other params

**Comment by macdonst** [comment]
@nhoctanker currently this plugin does not expose a way of calling `setContentInfo` on Android. What would be the use case for this? I'm not even sure where on the notification that text would be shown. Can you point it out on https://developer.android.com/design/patterns/notifications.html

**Comment by danieka** [comment]
It seems it's a deprecated method: https://developer.android.com/reference/android/app/Notification.Builder.html#setContentInfo(java.lang.CharSequence)
@nhoctanker do you still need this feature?

**Comment by macdonst** [comment]
@danieka yeah, I'd be dis-inclined to add this to the plugin now that the API is deprecated.
